tests/puppeteer/campaigns/functional/BO/catalog/products/filterAndQuickEditProducts.js

```diff
@@ -0,0 +1,212 @@
+      const numberOfProductsAfterReset = await this.pageObjects.productsPage.resetAndGetNumberOfLines();
+      await expect(numberOfProductsAfterReset).to.equal(numberOfProducts);
+    });
+
+    it('should filter by Name \'Customizable mug\'', async function () {
+      await this.pageObjects.productsPage.filterProducts('name', Products.demo_14.name);
+      const numberOfProductsAfterFilter = await this.pageObjects.productsPage.getNumberOfProductsFromList();
+      await expect(numberOfProductsAfterFilter).to.be.below(numberOfProducts);
+      for (let i = 1; i <= numberOfProductsAfterFilter; i++) {
+        const textColumn = await this.pageObjects.productsPage.getProductNameFromList(i);
+        await expect(textColumn).to.contains(Products.demo_14.name);
+      }
+    });
+
+    it('should reset all filters', async function () {
+      const numberOfProductsAfterReset = await this.pageObjects.productsPage.resetAndGetNumberOfLines();
+      await expect(numberOfProductsAfterReset).to.equal(numberOfProducts);
+    });
+
+    it('should filter by reference', async function () {
+      await this.pageObjects.productsPage.filterProducts(
+        'reference',
+        Products.demo_3.reference,
+      );
+      const numberOfProductsAfterFilter = await this.pageObjects.productsPage.getNumberOfProductsFromList();
+      await expect(numberOfProductsAfterFilter).to.be.below(numberOfProducts);
+      for (let i = 1; i <= numberOfProductsAfterFilter; i++) {
+        const textColumn = await this.pageObjects.productsPage.getProductReferenceFromList(i);
+        await expect(textColumn).to.contains(Products.demo_3.reference);
+      }
+    });
+
+    it('should reset all filters', async function () {
+      const numberOfProductsAfterReset = await this.pageObjects.productsPage.resetAndGetNumberOfLines();
+      await expect(numberOfProductsAfterReset).to.equal(numberOfProducts);
+    });
+
+    it('should filter by Category \'Art\'', async function () {
+      await this.pageObjects.productsPage.filterProducts(
+        'name_category',
+        Products.demo_5.category,
+      );
+      const numberOfProductsAfterFilter = await this.pageObjects.productsPage.getNumberOfProductsFromList();
+      await expect(numberOfProductsAfterFilter).to.be.below(numberOfProducts);
+      for (let i = 1; i <= numberOfProductsAfterFilter; i++) {
+        const textColumn = await this.pageObjects.productsPage.getProductCategoryFromList(i);
+        await expect(textColumn).to.contains(Products.demo_5.category);
+      }
+    });
+
+    it('should reset all filters', async function () {
+      const numberOfProductsAfterReset = await this.pageObjects.productsPage.resetAndGetNumberOfLines();
+      await expect(numberOfProductsAfterReset).to.equal(numberOfProducts);
+    });
+
+    it('should filter by Price(tax excl.) MIN-MAX \'13-20\'', async function () {
+      await this.pageObjects.productsPage.filterPriceProducts(13, 20);
```

Comment:
Maybe we can add prices in demo/products and use them here
```suggestion
      await this.pageObjects.productsPage.filterPriceProducts(Products.demo_1.price, Products.demo_2.price);
```